python/interpret_community/widget/explanation_dashboard.py

```diff
@@ -182,7 +188,7 @@ def get_base_url(self):
             # First handle known cloud environments
             if result is None:
                 # special case azure, since the azure sdk can set this env setting on local runs
-                if not in_cloud_env or "azureml_vm" in detected_envs:
+                if not in_cloud_env or self.env == CREDENTIALED_VM or self.env == PUBLIC_VM:
```

Comment:
Consider creating a set of `{CREDENTIALED_VM, PUBLIC_VM}` if they are often used together

Reply:
done!